### DISCUSSION TITLE
How to pull models after 0.4.0 update?

### DISCUSSION THREAD

**Discussion by Dima-Kal** [open-webui/open-webui#7079] [Q&A] [answered]
this is pre 0.4.0 screen with ability to pull models:
![Screenshot 2024-11-20 at 12 14 14](https://github.com/user-attachments/assets/839510b0-d1c6-4f90-95f8-75291002f4d7)
and this is new interface without any option to pull model:
![Screenshot 2024-11-20 at 12 16 51](https://github.com/user-attachments/assets/a0bbc30f-7d08-493b-92c6-6c7b9679bb31)
 searched every option and can't find it, also not documented in docs...

**Comment by DestroyCom** [comment, marked as answer]
https://github.com/open-webui/open-webui/discussions/7054#discussioncomment-11315403

I found the answer here, I don't know why they hid it there, I spent 30 minutes trying to find out where it was.

**Reply by Dima-Kal** [reply]
holy f***, that is odd choice to change the interface like that. thanks!
